app/models/ems_event.rb

```diff
@@ -130,9 +130,8 @@ def self.process_host_in_event!(event, options = {})
 
   def self.process_container_entities_in_event!(event, _options = {})
     [ContainerNode, ContainerGroup, ContainerReplicator].each do |entity|
-      process_object_in_event!(entity, event, :ems_ref_key => :ems_ref)
```

Comment:
I don't know the impact of the option deletion. An alternative could be:
 `process_object_in_event!(entity, event, :ems_ref_key => "{entity.name.underscore}_ems_ref".to_sym)`

Reply:
That's actually done automatically by `process_object_in_event!`, see https://github.com/agrare/manageiq/blob/afcec28ab2bb97b6f236e9219c6b66034b7b4e08/app/models/ems_event.rb#L94

Reply:
But was passing `:ems_ref_key => :ems_ref`, the the `ems_ref_key` property inside the same method will use the `key_prefix` only if `option[:ems_ref_key]` was `nil`, considering the attribution line:
 `ems_ref_key = options[:ems_ref_key] || "#{prefix}#{key_prefix}_ems_ref".to_sym`

Once `options[:ems_ref_key] ` will never be empty (because the :ems_ref_key is passed forward in the options), they will never use the  `entity.name.underscore` form, if I am not wrong

Reply:
Btw, just a curiosity, not a correction. It LGTM for me :smile:

Reply:
> Btw, just a curiosity, not a correction.

No no please keep it coming :)

`options[:ems_ref_key]` should always be nil because we aren't passing any `options` in.  This is covered in the spec tests https://github.com/ManageIQ/manageiq/pull/16076/files#diff-0ee77a30144f27bb7ba1f55b825ed746R37